.gitignore

```diff
@@ -1,2 +1,5 @@
 *.DS_Store
 *.html
+.Rproj.user
+attic/*
+*Rproj
```

Comment:
These are files to be ignired but that allow folks to work in an Rstudio project if they prefer